src/lib/ngsiNotify/senderThread.h

```diff
@@ -2,47 +2,63 @@
 #define SENDERTHREAD_H
```

Comment:
Could you elaborate on NOTIFICATION_WAIT_MODE, please? I think we introduced that in some point in the past time ago, but I don't see the "connection" with the cache stuff...